app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files = command.split(" ")
+    if files[0] != "cp" or files[1] == files[2]:
```

Comment:
Here you always have three words that make sense to unpack them into their own variables like `a, b, c = 1, 2, 3`